torch/testing/_internal/common_methods_invocations.py

```diff
@@ -10451,7 +10459,10 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
                DecorateInfo(unittest.expectedFailure, 'TestGradients', 'test_fn_gradgrad'),
                DecorateInfo(unittest.skip('Passes on complex128 and float64 only'), 'TestGradients', 'test_fn_fwgrad_bwgrad'),
                # AssertionError: Tensor-likes are not close! (new_empty_strided.default)
-               DecorateInfo(unittest.skip("Expected: new_empty_strided is not comparable"), 'TestDecomp', 'test_comprehensive'),)),
+               DecorateInfo(unittest.skip("Expected: new_empty_strided is not comparable"), 'TestDecomp', 'test_comprehensive'),
+               DecorateInfo(
+                   unittest.skip("Some stride values write multiple values to the same location e.g. (1,1,1,1)"),
+                   'TestCommon', 'test_compare_cpu'),)),
```

Comment:
@lezcano this is the only op with inputs that can be non-deterministic

Reply:
Great, thanks! A good follow-up PR would be to see if you can generate inputs that do not yield non-deterministic behaviour. I wouldn't be surprised if that fixed a few of the other skip/errors this function has.

Reply:
I created a WIP here https://github.com/pytorch/pytorch/pull/85583